numba_cuda/numba/cuda/cudadrv/nvvm.py

```diff
@@ -369,37 +369,40 @@ def get_log(self):
     (9, 0),
     (10, 0),
     (10, 1),
+    (10, 3),
     (12, 0),
+    (12, 1),
 )
 
 
 # Maps CTK version -> (min supported cc, max supported cc) inclusive
 _CUDA_CC_MIN_MAX_SUPPORT = {
-    (11, 1): ((3, 5), (8, 0)),
-    (11, 2): ((3, 5), (8, 6)),
-    (11, 3): ((3, 5), (8, 6)),
-    (11, 4): ((3, 5), (8, 7)),
-    (11, 5): ((3, 5), (8, 7)),
-    (11, 6): ((3, 5), (8, 7)),
-    (11, 7): ((3, 5), (8, 7)),
-    (11, 8): ((3, 5), (9, 0)),
-    (12, 0): ((5, 0), (9, 0)),
-    (12, 1): ((5, 0), (9, 0)),
-    (12, 2): ((5, 0), (9, 0)),
-    (12, 3): ((5, 0), (9, 0)),
-    (12, 4): ((5, 0), (9, 0)),
-    (12, 5): ((5, 0), (9, 0)),
-    (12, 6): ((5, 0), (9, 0)),
-    (12, 8): ((5, 0), (12, 0)),
+    (11, 1): [((3, 5), (8, 0)),],
+    (11, 2): [((3, 5), (8, 6)),],
+    (11, 3): [((3, 5), (8, 6)),],
+    (11, 4): [((3, 5), (8, 7)),],
+    (11, 5): [((3, 5), (8, 7)),],
+    (11, 6): [((3, 5), (8, 7)),],
+    (11, 7): [((3, 5), (8, 7)),],
+    (11, 8): [((3, 5), (9, 0)),],
+    (12, 0): [((5, 0), (9, 0)),],
+    (12, 1): [((5, 0), (9, 0)),],
+    (12, 2): [((5, 0), (9, 0)),],
+    (12, 3): [((5, 0), (9, 0)),],
+    (12, 4): [((5, 0), (9, 0)),],
+    (12, 5): [((5, 0), (9, 0)),],
+    (12, 6): [((5, 0), (9, 0)),],
+    (12, 8): [((5, 0), (10, 2)), ((12, 0), (12, 0)),],
```

Comment:
Have restructured how this mapping works to include a range of ranges. That way CUDA 12.8 is composed of two ranges:

1. A range from 5.0-10.2
2. A range from 12.0-12.0

As ranges are inclusive in the logic already, this picks up all CUDA compute capabilities for CUDA 12.8. While ignoring 10.3, which is only added in CUDA 12.9

Reply:
In case there is a CC 10.2 in the future, I think we should change the 12.8 supported toolkits not to include it:

```suggestion
    (12, 8): [((5, 0), (10, 1)), ((12, 0), (12, 0)),],
```